openeo/Pre&PostCovid.ipynb

```diff
@@ -0,0 +1,358 @@
+      "text/plain": [
+       "<Figure size 640x480 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax1 = plt.subplots(dpi=100)\n",
+    "ax1.plot(predata.t, predata.NO2.to_numpy().flatten(), color=\"g\")\n",
+    "ax1.set_xlabel(\"During COVID\")\n",
+    "ax2 = ax1.twiny()\n",
+    "ax2.plot(postdata.t, postdata.NO2.to_numpy().flatten(), color=\"r\")\n",
+    "ax2.set_xlabel(\"Post COVID\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f459e68-b702-4dd8-a07b-79d5f285b6b9",
+   "metadata": {},
+   "source": [
+    "The red line in the post illustrates the NO2 concentration duration during COVID, and the green line represents the period after COVID for similar months of various years. Although the shift is not substantial, there's a minor decline in NO2 levels. \n",
+    " \n",
+    "Thus, as a reference to this notebook, additional scenarios can be explored using the Sentinel 5P data within the Copernicus Dataspace Ecosystem.\n",
+    "\n",
+    "The red line in the plot shows the NO2 concentration during the COVID period over the selected region in Delhi, while the green line represents the post-COVID period for equivalent months in various years. It shows that although the shift is not substantial, there's a minor decline in NO2 levels.\n",
+    "Thuslytaking this nonotebo as a reference caseok, further scenarios can be investigated using Sentinel 5P data within the Copernicus Data Space Ecosyst, such as PM2.5 concentration, ozone layer depletion, SO2 concentration, etcem."
```

Comment:
this last paragraph looks bad

Reply:
Ahh my bad, for some reason, it looked fine in markdown but not after executing the cell. It has been corrected